StudyBuddy/Views/Profile/DisplayProfiles.cshtml

```diff
@@ -44,6 +44,7 @@
                         <ul class="list-group list-group-flush mt-4">
                             <li class="list-group-item">
                                 <strong>Birthday:</strong> @user.Traits.Birthdate.ToShortDateString()
+                                <strong>Location:</strong> @(string.IsNullOrEmpty(user.Traits.Location) ? "N/A" : user.Traits.Location)
```

Comment:
If the string is null, then the `user.Traits.Location` should already contain "N/A". This would introduce unnecessary variables.